docs/topics/shared-messaging-work-queue-mission-intro-paragraph.adoc

```diff
@@ -0,0 +1,8 @@
+Mission proficiency level:
```

Comment:
Difference between `docs/topics/messaging-work-queue-mission-intro-paragraph.adoc` and `docs/topics/shared-messaging-work-queue-mission-intro-paragraph.adoc`?